napari_skimage_regionprops/_regionprops.py

```diff
@@ -184,3 +196,371 @@ def ellipsoid_axis_lengths(table):
 
 regionprops_table_all_frames = analyze_all_frames(regionprops_table)
 register_function(regionprops_table_all_frames, menu="Measurement > Regionprops of all frames (nsr)")
+
+
+def make_element_wise_dict(list_of_keys, list_of_values):
+    """Return an element-wise dictionary of two lists.
+
+    From a list of keys and a list of values, return a dictionnary linking
+    those lists. Both lists must have the same length.
+
+    Parameters
+    ----------
+    list_of_keys : List
+        list to be used as dictionnary keys.
+    list_of_values : List
+        list to be used as dictionnary values.
+
+    Returns
+    -------
+    Dictionnary
+    """
+    return dict(
+                list(
+                    map(
+                        lambda list_of_keys, list_of_values:
+                        [list_of_keys, list_of_values],
+                        *[list_of_keys, list_of_values]
+                        )
+                    )
+                )
+
+
+@register_function(
+    menu="Measurement > Regionprops map multichannel (scikit-image, nsr)")
+def napari_regionprops_map_channels_table(
+        label_images: List[napari.types.LabelsData],
+        intensity_images: List[napari.types.ImageData],
+        reference_label_image: napari.types.LabelsData,
```

Comment:
I propose this parameter first as it is most important. The other labels and images are optional

Reply:
I agree, the `label_image_reference` is now the first and mandatory parameter.
For the napari function that yields the widget, the other labels and images are optional, but I was unable to assign initial values to them and still maintain the proper widget generation.
I don't think this is a problem because a user would use one of the 4 individual functions instead. I wrote notebooks explaining how to use it via plugin and via code.